src/index.html

```diff
@@ -11,7 +11,223 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form onsubmit="onSubmit()" action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post">
+
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <label  for="surname">
+          Surname:
+          </label>
+          <input
+          type="text"
+          name="Surname"
+          id="surname"
+          required
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="Surname"
            id="surname"
            required
          >
```
Давай атрибутам отступ в два пробела